experimental/dds/tree/src/id-compressor/IdCompressor.ts

```diff
@@ -654,7 +654,7 @@ export class IdCompressor {
 						// the finalization of the ranges.
 						this.sessionIdNormalizer.addFinalIds(finalPivot, newLastFinal, currentCluster);
 						this.logger?.sendTelemetryEvent({
```

Comment:
should all of these telemetry calls in this method be wrapped in an if(isLocal)?

Reply:
if not, all clients will send them for all ranges

Reply:
This one's already in an `if (isLocal)` block above so we should be good to go here.

Reply:
oops, good call!